packages/storage-postgres/migrations/0055_scheduled_item_recurrence_reminders.sql

```
@@ -0,0 +1,29 @@
+--> statement-breakpoint
+ALTER TABLE "space_scheduled_items" ADD COLUMN "matrix_room_id" text;
+--> statement-breakpoint
+ALTER TABLE "space_scheduled_items" ADD COLUMN "matrix_auto_link" boolean DEFAULT false NOT NULL;
+--> statement-breakpoint
+ALTER TABLE "space_scheduled_items" ADD COLUMN "remind_email" boolean DEFAULT false NOT NULL;
+--> statement-breakpoint
+ALTER TABLE "space_scheduled_items" ADD COLUMN "remind_push" boolean DEFAULT false NOT NULL;
+--> statement-breakpoint
+ALTER TABLE "space_scheduled_items" ADD COLUMN "reminder_minutes_before" integer;
+--> statement-breakpoint
+CREATE INDEX "space_scheduled_items_reminder_idx" ON "space_scheduled_items" USING btree ("remind_email","remind_push","reminder_minutes_before");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)